src/adapters/scene-fetcher.ts

```diff
@@ -84,6 +86,12 @@ export async function createSceneFetcherComponent(
     return permissionsCache.fetch(`${catalystContentUrl}/contents/${sceneId}`)
   }
 
+  async function fetchOnWorldActionPermissions(worldName: string): Promise<PermissionsOverWorld | undefined> {
+    const response = await fetch.fetch(`${worldContentUrl}/world/${worldName}/permissions`)
+    const data = await response.json()
+    return data.permissions
+  }
```

Comment:
Could it happen that the world doesn't exist or the request fails because of something else? Should we handle those cases?